### ISSUE TITLE
Datepicker doesn't work with jQuery 3.3.1

### ISSUE THREAD

**Issue by DSpade-r** [wenzhixin/bootstrap-table#3712]
I catch the error: Syntax error, unrecognized expression: #

**Comment by djhvscf** [comment]
Please provide a [jsFiddle](https://github.com/wenzhixin/bootstrap-table/blob/develop/CONTRIBUTING.md#bug-reports) to show your problem, thanks!

**Comment by DSpade-r** [comment]
Yes, I did approximately the real project. With your framework I met only 4 days ago, so maybe not all understand in detail. [https://jsfiddle.net/b5r3hn5L/29/](url). jFiddle does not show errors. I need to filter by date (preferably between two dates), but I can not do the simplest example.

**Comment by djhvscf** [comment]
The example provided does not have any error. @agmorpheus could you provide a jsfiddle?

**Comment by agmorpheus** [comment]
Here an example: https://jsfiddle.net/d0e8vjqh/3/
